.vscode/settings.json

```diff
@@ -0,0 +1,36 @@
+{
+  "javascript.preferences.quoteStyle": "double",
+  "typescript.preferences.quoteStyle": "double",
+  "javascript.preferences.importModuleSpecifier": "shortest",
+  "typescript.preferences.importModuleSpecifier": "shortest",
+  "javascript.updateImportsOnFileMove.enabled": "always",
+  "typescript.updateImportsOnFileMove.enabled": "always",
+  "editor.detectIndentation": true,
+  "eslint.format.enable": true,
+  "eslint.run": "onType",
+  "[javascript]": {
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": true
+    }
```

Comment:
Ahh, so here's a difference. I let eslint organize the imports instead of vscode. I'm not sure if they sort the same way. We already have an `'import/order'` rule in the .eslintrc file.

Reply:
VS Code actually does slightly better when it comes to organizing, but it doesn't do any grouping like eslint. They don't clash at least. 🤷

Reply:
Not all of us are strictly using vscode, either, so it should respect whatever eslint is doing because if we `eslint --fix` in the future it will change things around.

Reply:
True. I'll turn it off.